### PR TITLE
macos arm64 ci

### PR DESCRIPTION
**Pull request by mwestphal**
fix: #1240

### CODE REVIEW

**Comment by Meakk** [comment]
I'm posting the failing images here:

|Test|Result|Diff|
|--|--|--|
|TestConfigStemBuild|TestConfigStemBuild|TestConfigStemBuild diff|
|TestDXF|TestDXF|TestDXF diff|
|TestGridX|TestGridX|TestGridX diff|
|TestGridZ|TestGridZ|TestGridZ diff|

**Comment by mwestphal** [comment]
test that fails systematically:

	 51 - f3d::TestGridX
	132 - f3d::TestConfigFileBuild
	133 - f3d::TestConfigStemBuild

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -145,17 +145,50 @@ jobs:
...
+  macos_arm:
+    if: github.event.pull_request.draft == false
+    needs: cache_lfs
+
+    strategy:
+      fail-fast: false
+      matrix:
+        vtk_version: [commit]
+        cpu: [arm64]
+        bundle_label: [no-bundle]
```
Comment (Meakk) [inline .github/workflows/ci.yml:185]: Can you create a bundle/no-bundle matrix instead of the include field here?

Comment (mwestphal) [reply .github/workflows/ci.yml:185]: I think I should still test 9.3.0 and 9.2.6, I forgot to add them back.

I can remove bundle if you want though.

**Review by Meakk — APPROVED** [review]

**Pull request merged**